packages/arb-token-bridge-ui/src/components/common/HeaderMenu.tsx

```diff
@@ -90,6 +97,10 @@ export function HeaderMenuMobile(
             >
               {props.children}
             </span>
+
+            {props.items?.length && (
+              <ChevronDownIcon className="ml-1 h-4 w-4 shrink-0 grow-0 text-white" />
```

Comment:
Maybe instead of setting a margin here we could add a `space-x-1` class on the container

Reply:
I originally tried to use `space-x-1` but for some reason, it doesn't work. Maybe the conditional `lg:inline-flex` is not conducive. `ml-1` works well. Let me know if I need to look deeper into this.